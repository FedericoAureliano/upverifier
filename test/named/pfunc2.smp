type [k, v] map := Array[k, v]

function [v] fill (default: v) : map[Int, v] := const(default, map[Int, v])

print(fill(0))
print(fill("test"))