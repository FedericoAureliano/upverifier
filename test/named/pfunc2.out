const(0, Array[Int, Int])
const(test, Array[Int, Int])
